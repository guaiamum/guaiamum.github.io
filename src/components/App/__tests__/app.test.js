import { h } from 'preact';
import { shallow } from 'enzyme';

import App from '../index';

jest.mock('Components/Profile', () => (<h1>Profile</h1>));
jest.mock('Components/Main', () => (<h2>Main</h2>));

describe('App', () => {
    it('renders Profile and Main', () => {
        const wrapper = shallow(<App />);
        expect(wrapper).toMatchSnapshot();
    });
});
