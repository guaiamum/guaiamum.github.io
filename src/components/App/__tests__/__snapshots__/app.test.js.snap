// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders Profile and Main 1`] = `ShallowWrapper {}`;
